src/t8_forest/t8_forest_general.h

```diff
@@ -571,6 +571,11 @@ t8_forest_leaf_face_neighbors (t8_forest_t forest, t8_locidx_t ltreeid, const t8
                                t8_locidx_t **pelement_indices, t8_eclass_scheme_c **pneigh_scheme,
                                int forest_is_balanced);
 
+/** Like \ref t8_forest_leaf_face_neighbors but also provides information about the global neighbors and the orientation. 
+  * \param [out]   gneigh_tree  The global tree IDs of the neighbor trees.
+  * \param [out]   orientation  The face orientation is computed and stored there.
```

Comment:
```suggestion
  * \param [out]   orientation  If not NULL on input, the face orientation is computed and stored here. 
  *                                         Thus, if the face connection is an inter-tree connection the orientation of the tree-to-tree connection is stored. 
  *                                         Otherwise, the value 0 is stored.
```